test/src/service/filtering/filters_test.py

```diff
@@ -132,6 +134,51 @@ def test_rangefilter_from_string_fail():
             RangeFilter.from_string(type_, input_)
 
 
+@pytest.fixture
+def boolean_filter_true():
+    return BooleanFilter(True)
+
+
+@pytest.fixture
+def boolean_filter_false():
+    return BooleanFilter(False)
+
+
+def test_boolean_filter_equality(boolean_filter_true, boolean_filter_false):
+    assert boolean_filter_true == boolean_filter_true
+    assert boolean_filter_false == boolean_filter_false
+    assert boolean_filter_true != boolean_filter_false
+    assert boolean_filter_true != RangeFilter(ColumnType.FLOAT, -56.1, 1913.1, True, True)
+
+
+def test_boolean_filter_representation(boolean_filter_true, boolean_filter_false):
+
+    assert repr(boolean_filter_true) == "BooleanFilter(True)"
+    assert repr(boolean_filter_false) == "BooleanFilter(False)"
+
+    assert eval(repr(boolean_filter_true)) == boolean_filter_true
+    assert eval(repr(boolean_filter_false)) == boolean_filter_false
+
+
+def test_boolean_filter_from_string():
+    assert BooleanFilter.from_string(None, "true") == BooleanFilter(True)
+    assert BooleanFilter.from_string(None, "false") == BooleanFilter(False)
+    with raises(errors.IllegalParameterError):
+        BooleanFilter.from_string(None, "invalid")
+
+
+def test_boolean_filter_to_arangosearch_aql(boolean_filter_true, boolean_filter_false):
+    identifier = "test_identifier"
+    var_prefix = "test_prefix"
```

Comment:
I would
a) make these different for the true and false cases, and
b) inline them to reduce recapitulation of the code under test

Reply:
👍

Reply:
:+1: